myco_api_client/matchers/myco_matcher_client_interface.py

```diff
@@ -59,3 +59,6 @@ def on_finish(self, data: Dict):
 
     def on_event_or_response(self, data: Dict):
         """Extra handler for all events/responses callbacks."""
+
+    def on_area_map(self, data: Dict):
```

Comment:
This is not an event but a command response, therefore I would rename it to `on_area_map_response`